map_server:
  ros__parameters:
    yaml_filename: "maps/default_map.yaml"
    save_map_timeout: 5.0

map_saver:
  ros__parameters:
    use_sim_time: False
    save_map_timeout: 5.0
    free_thresh_default: 0.25
    occupied_thresh_default: 0.65
    map_subscribe_transient_local: True
